src/springfield/fields.py

```diff
@@ -499,7 +498,7 @@ def _resolve_dotted_name(dotted_name):
         except Exception:
             raise_(
                 ValueError,
-                'Invalid class name for EntityField: %s' % dotted_name,
+                ValueError('Invalid class name for EntityField: %s' % dotted_name),
```

Comment:
`future.utils.raise_` worked the other way, but `six.reraise` used here expects the second arg to be an exception.

Reply:
`raise_from(ValueError(...), e)` would be better for Python 3.